workshop_app/templates/workshop_app/propose_workshop.html

```diff
@@ -1,63 +1,71 @@
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
-    <script src="{{URL_ROOT}}/static/workshop_app/js/bootstrap-3.3.7.min.js"></script>
-
-    <!-- calender -->
     <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js"></script>
 
     <script>
-    var dateToday = new Date();
-    var upto = new Date();
-
-    dateToday.setDate(dateToday.getDate() + 3);
-    upto.setFullYear(dateToday.getFullYear() + 1);
+        var dateToday = new Date();
+        var upto = new Date();
 
-      $(function() {
-        $( ".datepicker" ).datepicker({
-          changeMonth: true,
-          changeYear: true,
-          minDate: dateToday,
-          maxDate: upto,
-          beforeShowDay: $.datepicker.noWeekends,
+        dateToday.setDate(dateToday.getDate() + 3);
+        upto.setFullYear(dateToday.getFullYear() + 1);
 
-          // You can put more options here.
+        $(function () {
+            $(".datepicker").datepicker({
+                changeMonth: true,
+                changeYear: true,
+                minDate: dateToday,
+                maxDate: upto,
+                beforeShowDay: $.datepicker.noWeekends,
+            });
         });
-      });
-  </script>
-  {% endblock %}
+    </script>
+{% endblock %}
 
 
 {% block content %}
-  <style type="text/css">
-      .errorlist {
-      color: red;
-    }
-  </style>
-
-  <div class="container">
-	<div class="alert alert-info">
-  		Please Note: Students have to take a mandatory test before Advanced Python
-		Workshop only those who will clear this test will be eligible to attend
-		the workshop.
-	</div>
-    <form method="post">
-      {% csrf_token %}
-      <br/>
-      {{ form.as_p }}
-      <br><br>
-      <button class="btn btn-success" type="submit">Submit</button>
-    </form>
-    <br>
-
-  </div>
+    <style type="text/css">
+        .errorlist {
+            color: red;
+        }
+    </style>
+
+    <div class="container" align="center">
+        <div class="alert alert-info">
+            Please Note: Students have to take a mandatory test before Advanced Python
+            Workshop only those who will clear this test will be eligible to attend
+            the workshop.
+        </div>
+        <br><br>
+        <div class="col-md-6 col-md-offset-6">
+            <div class="card border-primary">
+                <div class="card-header">
+                    <h3>Propose Workshop</h3>
+                </div>
+                <div class="card-body">
+                    <div class="form-group">
+                        <form method="post">
+                            {% csrf_token %}
+                            <br/>
+                            {{ form.workshop_type }}
+                            <br>
+                            {{ form.date }}
+                            <br>
+                            {{ form.tnc_accepted }} I accept the <a href="/tnc">terms and conditions</a>
```

Comment:
Show the terms and conditions of the workshop type.

Reply:
Will be fixing later